crates/walrus-sui/src/client/retry_client/retriable_rpc_error.rs

```diff
@@ -8,7 +8,7 @@ use std::fmt::Debug;
 use super::{super::SuiClientError, CheckpointRpcError, FallbackError, RetriableClientError};
 
 /// The list of HTTP status codes that are retriable.
-const RETRIABLE_RPC_ERRORS: &[&str] = &["429", "500", "502"];
+const RETRIABLE_RPC_ERRORS: &[&str] = &["408", "429", "500", "502", "504"];
```

Comment:
Maybe add 503 as well?
```suggestion
const RETRIABLE_RPC_ERRORS: &[&str] = &["408", "429", "500", "502", "503", "504"];
```

Reply:
I haven't seen it. But according to the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/503), this should be used for a temporary outage, for example due to upgrades, maintenance, etc. So it makes sense to retry this IMO.